incubator/kafka/templates/StatefulSet.yaml

```diff
@@ -1,55 +1,3 @@
-# headless service for petset DNS
-apiVersion: v1
-kind: Service
-metadata:
-  name: "{{ printf "%s-zookeeper-%s" .Release.Name .Values.zookeeper.Name | trunc 63 }}"
-spec:
-  ports:
-  - port: 2181
-    name: client
-  selector:
-    component: "{{.Release.Name}}"
-
-# A headless service to create DNS records
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: "{{ printf "%s-broker-%s" .Release.Name .Values.Name | trunc 63 }}"
-  labels:
-    heritage: {{.Release.Service | quote }}
-    release: {{.Release.Name | quote }}
-    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
-    component: "{{.Release.Name}}-{{.Values.Component}}"
-  annotations:
-    "helm.sh/created": {{.Release.Time.Seconds | quote }}
-    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
-spec:
-  ports:
-  - port: 9092
-  clusterIP: None
-  selector:
-    component: "{{.Release.Name}}-{{.Values.Component}}"
-    app: kafka
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: "{{ printf "%s-%s" .Release.Name .Values.Name | trunc 63 }}"
-  labels:
-    heritage: {{.Release.Service | quote }}
-    release: {{.Release.Name | quote }}
-    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
-    component: "{{.Release.Name}}-{{.Values.Component}}"
-  annotations:
-    "helm.sh/created": {{.Release.Time.Seconds | quote }}
-spec:
-  ports:
-  - port: 9092
-  selector:
-    component: "{{.Release.Name}}-{{.Values.Component}}"
-    app: kafka
----
```

Comment:
@jrnt30 now both charts (`zookeeper` and `kafka`) are providing a Service (VIP) for _zookeeper_ clients to access the _zookeeper_ cluster. This is redundant and I think we can get rid of the one provided by _kafka_. What do you think about using a single Service VIP which will load balance among the _zookeeper_ backends?